tests/test_copy.py

```diff
@@ -20,11 +20,11 @@ def test_bracelabel_copy(tmp_path):
     # For this test to work, we need to tweak some folders temporarily
     original_text_dir = file_writer_config["text_dir"]
     original_tex_dir = file_writer_config["tex_dir"]
-    mediadir = os.path.join(tmp_path, "deepcopy")
-    file_writer_config["text_dir"] = os.path.join(mediadir, "Text")
-    file_writer_config["tex_dir"] = os.path.join(mediadir, "Tex")
+    mediadir = Path(tmp_path).joinpath("deepcopy")
```

Comment:
```suggestion
    mediadir = Path(tmp_path) / "deepcopy"
```
and for the others as well